.pipelines/vhd-builder.yaml

```diff
@@ -10,7 +10,7 @@ variables:
   CONTAINER_IMAGE:  'mcr.microsoft.com/oss/azcu/go-dev:v1.32.3'
 
 pool:
-  vmImage: 'Ubuntu 18.04'
+  vmImage: 'Ubuntu 20.04'
```

Comment:
We have been told that pools should be ubuntu-latest but if this works, that may be fine.

Note that the pool image should not impact the builder image (below)